### ISSUE TITLE
image in 'card' cannot have control on the size

### ISSUE THREAD

**Issue by khoivan88** [saadeghi/daisyui#181]
Hi, sorry for another question, I was trying to control the size of image inside a card, however, regardless of where I put `w-56 max-w-md`, the size of the image seems to always 100% the image size? I used the Chrome inspection and it seems that the card default is always 100% and it overrides other classes 

Maybe I did something wrong?

Example: 

```html
<div class="container mx-auto my-2 md:my-5 card lg:card-side lg:space-x-12">
    <figure class="mx-auto max-w-max">
    <img src="/team/john-doe/john-doe.jpg" alt="Picture of John Doe" width="" height="" loading="lazy" class="w-24 max-w-md rounded-lg">
  </figure>
    <div class="card-body space-y-4">
        <p class="leading-loose">John grew up in Las Vegas, Nevada,...</p>
    </div>
</div>
```

**Comment by saadeghi** [comment]
Related to https://github.com/saadeghi/daisyui/issues/30

**Comment by saadeghi** [comment]
Fixed in v2.0
